src/pathfinding_service/database.py

```diff
@@ -216,7 +218,9 @@ def get_feedback_token(
 
         if token:
             return FeedbackToken(
-                token_network_address=decode_hex(token["token_network_address"]),
+                token_network_address=TokenNetworkAddress(
+                    decode_hex(token["token_network_address"])
```

Comment:
Above we use `TokenNetworkAddress(to_canonical_address(...))`. Using the same would be more consistent.

Reply:
True, I missed that during my automatic replace.